### PR TITLE
Readability and Keywords are added.

### PR DESCRIPTION
**Pull request by vrkansagara**
I have add more readability in following things.
- [x] LICENSE.md and LICENCSE.txt files.
- [x] Moved license file to root folder which is general but obvious.
- [x] Build status icon moved to the top of the README.md file.

### CODE REVIEW

**Comment by sergeyklay** [comment]
We don't add changes directly to stable branches or master. Also can you please explain a bit more which issue aims to solve this PR?

**Comment by vrkansagara** [comment]
@sergeyklay  I dont think that this PR is aims to any issue this is general improvement for the documentation and more clear navigation for the project and other crawler and human readable so.

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
-language: php
+        language: php
 
 php:
-  - 5.3
```
Comment (Green-Cat) [inline .travis.yml]: Already done in the 2.1.x branch.

File: composer.json [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
+	"name" : "phalcon/cphalcon",
+	"description" : "Web framework delivered as a C-extension for PHP",
+	"keywords" : [
+		"extension",
+		"PHP",
+		"C-extension",
+		"framework"
+	],
+	"require" : {
+		"php" : "~5.4"
```
Comment (Green-Cat) [inline composer.json]: Already done in 2.1.x

Comment (vrkansagara) [inline composer.json]: @Green-Cat  thanks :+1:

File: .travis.yml (hunk 1 above)
Comment (vrkansagara) [inline .travis.yml]: @Green-Cat  Thanks :+1:

**Pull request closed without merging**